### ISSUE TITLE
[Bug] Walk on powder snow causes fps drops 

### ISSUE THREAD

**Issue by DankestO** [alRex-U/ParCool#284]
**Summary**
Hello good afternoon, It seems that Parcool does something with the Iron's Spellbooks curios equipment, I come from there to report it here.
https://github.com/iron431/irons-spells-n-spellbooks/issues/559

**Running Environment**
+ Minecraft Version: 1.20.1
+ ParCool Version: 1.20.1-3.2.1.2-R.
+ Forge Version (optional): 1.20.1 - 47.2.00

**Detail**
As I was testing, this happens when I equip Iron's Spellbooks rings and fall in powder snow, with the others Curios slots equipped I had no problems, the more rings I have equipped, the more fps I lose, to the point of being unplayable when I have all 4 and start falling through the snow.

**Additional Data**
I hope the data from Spark helps you:
https://spark.lucko.me/OH1RA2fCvB

**Comment by alRex-U** [comment]
Thanks for reporting.
Do you have any videos showing it?

**Comment by DankestO** [comment]
Sure, https://youtu.be/M7MJ-hlAv0M

I have to mention that I recorded this on a dedicated server, when I tried it in singleplayer the fps didn't drop that much, also the rings are improved with Apotheosis, but I haven't noticed any changes between improved or unimproved Iron's Spell rings.

It only happens with the curious Iron's Spellbook equipment, without armor and testing the rings it also happens.

**Comment by DankestO** [comment]
I created another new server to test something else, but I found that on this new server with similar items and same seed this bug didn't occur, with or without parcool.

I was trying for a while different combinations or upgrades to add to the equipment. But it seems that nothing happened.
It's strange, so this only happens with the main server that we have with some friends, with the same mods as with this test server.

I'll close this, I think this must be some strange mix of circumstances that don't happen normally.
